content/en/_redirects

```diff
@@ -184,7 +184,7 @@ docs/started/requirements/                    /docs/started/getting-started/
 /docs/components/pipelines/sdk/v2/*                           /docs/components/pipelines/sdk-v2/:splat
 
 # pipelines v1 -> v2 redirects
-/docs/components/pipelines/*        /docs/components/pipelines/v1/:splat
+/docs/components/pipelines/*        /docs/components/pipelines/legacy-v1/:splat
```

Comment:
@hbelmiro I am not sure this redirect is doing what you think it is.

When I go to `https://deploy-preview-3737--competent-brattain-de2d6d.netlify.app/docs/components/pipelines/v1/introduction/` I get a 404.

I end up on `https://deploy-preview-3737--competent-brattain-de2d6d.netlify.app/docs/components/pipelines/legacy-v1/v1/introduction/`.

We might honestly need to move everything to a `/pipelines-v2/` docs folder path and leave all the old redirects to make everything work properly.

Reply:
I see. Good catch @thesuperzapper.

Maybe we should bring the v1 and v2 folders back to avoid facing the same issue in future versions (v3, v4...).
WDYT @StefanoFioravanzo?

Reply:
@hbelmiro We are in an impossible situation with the `/pipelines` prefix, as doing anything else under there will break old redirects, the goal is that any old page location will chain redirects to the eventual new home.

Because we want to reduce nesting, I don't like that we currently have V1/V2 directly under pipelines. From a user-facing perspective, I love the "Legacy V1" section (note: we should remove the `-` in the name using an `_index.md` in the folder's root.

So the question becomes how to achieve this, and I think the only way would be to create a new `/pipelines-new/` section which uses the same structure you have now, but redirects from the old `/pipelines/v1/` and `/pipelines/v2/` correctly.

Reply:
@thesuperzapper I found a way to make the redirects work while keeping the structure. But I had to move the redirects upper, otherwise they don't work. It seems like they are replaced.

Reply:
@hbelmiro It's because of the catch-all redirects that exist below yours, by putting explicit redirects for all pages you avoid those.

My recommendation is we just remove the catch-all ones, and make sure that at least all the __current pages__ have valid redirects. 

I checked all the old pages, and people messed up the redirects for them anyway, so we don't really lose anything.

See https://github.com/kubeflow/website/pull/3737#discussion_r1621338999

Reply:
Done.